codegen/smithy-aws-swift-codegen/src/test/kotlin/software/amazon/smithy/aws/swift/codegen/restxml/AWSRestXMLHttpResponseBindingErrorGeneratorTests.kt

```diff
@@ -106,12 +106,12 @@ class AWSRestXMLHttpResponseBindingErrorGeneratorTests {
                 public var _isThrottling: Swift.Bool = false
                 public var _type: ClientRuntime.ErrorType = .client
                 public var header: Swift.String?
-                public var nested: RestXmlClientTypes.ComplexXMLNestedErrorData?
+                public var nested: RestXmlerrorsClientTypes.ComplexXMLNestedErrorData?
```

Comment:
Name of this type changed because it matches the `sdkId` in this test's Smithy def.